glom/test/test_path_and_t.py

```diff
@@ -203,13 +205,22 @@ def test_path_star():
     assert glom(val, '**.c') == [1, 2, 3]
     assert glom(val, 'a.**.c') == [1, 2, 3]
     assert glom(val, 'a.*.b.*.c') == [[1, 2]]
+    core.PATH_STAR = False
 
 
 def test_star_broadcast():
+    core.PATH_STAR = True
     val = {'a': [1, 2, 3]}
     assert glom(val, Path.from_text('a.*').path_t + 1) == [2, 3, 4]
     val = {'a': [{'b': [{'c': 1}, {'c': 2}, {'c': 3}]}]}
     assert glom(val, Path.from_text('**.c').path_t + 1) == [2, 3, 4]
```

Comment:
do we also want to test `T.__starstar__().c`, or is `__starstar__()` not part of the public API?